src/daemon/Seat.cpp

```diff
@@ -141,4 +142,15 @@ namespace SDDM {
             VirtualTerminal::jumpToVt(*nextVt, true);
         }
     }
+
+    bool Seat::canTTY() {
+        OrgFreedesktopLogin1ManagerInterface manager(Logind::serviceName(), Logind::managerPath(), QDBusConnection::systemBus());
+        auto seatPath = manager.GetSeat(m_name);
+        OrgFreedesktopLogin1SeatInterface sessionIface(Logind::serviceName(), seatPath.value().path(), QDBusConnection::systemBus());
+        if (manager.isValid() && sessionIface.property("canTTY").isValid()) {
+            return sessionIface.canTTY();
+        } else {
+            return m_name.compare(QStringLiteral("seat0"), Qt::CaseInsensitive) == 0 && QFile::exists(VirtualTerminal::getDefaultVtPath());
```

Comment:
The method wouldn't be necessary, you can use `VirtualTerminal::defaultVtPath` directly. If you want to be fancy even the qstring can be avoided by doing `access(VirtualTerminal::defaultVtPath, F_OK) == 0`

Reply:
Do I need to include something additional?
Trying
`return m_name.compare(QStringLiteral("seat0"), Qt::CaseInsensitive) == 0 && access(VirtualTerminal::defaultVtPath, F_OK) == 0);`
I get
`src/daemon/Seat.cpp:153:96: No member named 'defaultVtPath' in namespace 'SDDM::VirtualTerminal'`

Reply:
Hm, is it missing a declaration in `VirtualTerminal.h`?

Reply:
I figured this one out